src/app.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { AppController } from './app.controller';
+import { PrismaService } from './database/prisma.service';
+
+@Module({
+  imports: [],
+  controllers: [AppController],
+  providers: [PrismaService],
```

Comment:
Consider adding `DatabaseModule` and adding / exporting `PrismaService` there. Then import `DatabaseModule` here in `AppModule`